.github/CONTRIBUTING.md

```diff
@@ -34,7 +34,10 @@ We have a handful of simple standards for commit messages which help us to gener
 - **3rd line:** `refs/closes #000` or `no issue`
 - **4th line:** Whatever you want. Any extra details can be included from here
 
-If your change is **user-facing** please prepend the first line of your commit with one of the following emoji keys, which are used in the changelog:
+If your change is **user-facing** please prepend the first line of your commit with **an emoji key**.
```

Comment:
I think now we're using emojis for all changes not just user facing? Maybe i am wrong?
```suggestion
Please prepend the first line of your commit with **an emoji key**.
```

Reply:
No, still only user-facing changes or large architectural changes that could have unintended consequences such as the switch in ghost-admin auth that we failed to highlight last cycle.

Reply:
oh right - i've been adding them to everything :see_no_evil: